routes/api/articles.js

```diff
@@ -148,4 +149,11 @@ router.post(
   asyncHandler(ArticleController.reportArticle)
 );
 
+router.post(
+  '/:articleSlug/comment-on-text',
```

Comment:
@NkFab `comment-on-text` is too long as a request param. Try to look for a single word such as `highlight`

Reply:
@oesukam highlight have a different meaning to what we have to achieve